### PR TITLE
Add account_service_log_out_url

### PR DESCRIPTION
**Pull request by insacuri**

### CODE REVIEW

File: authentication/auth.go [hunk 1]
```diff
@@ -314,9 +314,10 @@ func getStringOrDefault(key string, values map[string][]string, defaultValue str
 }
 
 // GenerateTokenFromDefaults coverts a set of DEFAULT values into a JWT
-func GenerateTokenFromDefaults(surveyURL string, accountServiceURL string, urlValues url.Values) (token string, error string) {
+func GenerateTokenFromDefaults(surveyURL string, accountServiceURL string, accountServiceLogOutURL string, urlValues url.Values) (token string, error string) {
 	claims := make(map[string]interface{})
 	urlValues["account_service_url"] = []string{accountServiceURL}
+	urlValues["account_service_log_out_url"] = []string{accountServiceLogOutURL}
```
Comment (jonnyshaw89) [inline authentication/auth.go:320]: Where is this coming from?
I cant see it in the docs https://ons-schema-definitions.readthedocs.io/en/latest/respondent_management_to_electronic_questionnaire.html

Comment (insacuri) [reply authentication/auth.go:320]: Getting it from ras.  Forgot to update the document

Comment (jonnyshaw89) [reply authentication/auth.go:320]: The doc should probably have been updated and agreed before we made this change.
As we should then be making changes to support the changed interface.

**Review by dwyeradam — APPROVED** [review]
On the assumption that the [Schema Definition PR](https://github.com/ONSdigital/ons-schema-definitions/pull/39) gets in.

**Review by bitdivision — APPROVED** [review]

**Comment by dwyeradam** [comment]
@insacuri if you squash this I don't see any issue with merging it before runner is ready to use it.

**Pull request merged**